integration/docker/Dockerfile

```diff
@@ -39,7 +39,10 @@ ARG ALLUXIO_UID=1000
 ARG ALLUXIO_GID=1000
 ARG ENABLE_DYNAMIC_USER=false
 
-RUN apk --no-cache --update add bash libc6-compat shadow tini && \
+USER root
```

Comment:
is there a reason to set the user as root? i would advise omitting unnecessary changes. changing users also increases the docker image size